workflows/DATA_TO_MODEL.nf

```diff
@@ -3,14 +3,18 @@
 nextflow.enable.dsl = 2
 // Empty string default to avoid warning
 params.submissions = ""
+// The expected file type for the submissions
+params.file_type = "csv"
```

Comment:
should this just be file extension (i.e. `".csv"`)? and if you want to prevent some issues/obviate some handling logic later you could make this lowercase using Nextflow logic.

Reply:
[Updated](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/pull/42/commits/6b7820c3cb65c96e8fef4487e7db7a8df495fcd2)! Lowercase is enforced now. I didn't change this to a file extension parameter (e.g. `.csv`) because I think adding non-alphabet characters to a parameter makes things confusing from a user perspective, but if you prefer I add this I can do that. LMK!

Reply:
Makes sense to me! Thanks